middlewares/validators/user.js

```diff
@@ -17,10 +16,8 @@ const updateUser = async (req, res, next) => {
     linkedin_id: joi.string().optional(),
     twitter_id: joi.string().optional(),
     instagram_id: joi.string().optional(),
-    site: joi.string().optional(),
-    isMember: joi.boolean().optional(),
-    tokens: joi.object().optional(),
-    website: joi.string().optional()
+    website: joi.string().optional(),
+    isMember: joi.boolean().optional()
```

Comment:
We don't need this here -- `isMember`
It should not be here from security pov
Otherwise users will mark themselves as members 🤣 

Please remove it from here

Reply:
oh yeahh! Okay